### PR TITLE
docs(explorations): graduate patent-citation-candor-gate into its goal packet

### PR DESCRIPTION
**Pull request by kriegcloud**
## Summary

Decomposes and graduates the `patent-citation-candor-gate` exploration (docs-only):

- **`MAP.md` (decompose, stage 4):** one candidate goal packet per the approved BRIEF's
  appetite, two strict rungs (rung 1 in-memory domain proof around a failing-then-green
  `CandorPolicy.test.ts`; rung 2 durability + the law-practice slice's first db-admin
  migration + live promotion-path invocation), a file-grounded capability check
  (every component a verified REUSE cite or explicit NET-NEW), and follow-on children
  named with triggers but NOT graduated.
- **Graduation (stage 5):** the four-point definition-of-ready passed, so
  `goals/patent-citation-candor-gate/` is scaffolded from `goals/_template`: SPEC seeded
  from the BRIEF (no-gos -> non-goals, rabbit holes -> constraints, DECISIONS -> SPEC
  decision log), PLAN with rung phases, GOAL launcher (3,676 chars), README,
  `initiative-manifest/v2` manifest with `provenance.exploration` wired, and the
  inherited `research/SOURCES.md` ledger.
- **Loop closed:** exploration manifest -> `graduate`/`graduated` + `links.goals`;
  packet README trail/next-question; DECISIONS decompose+graduate entry; ATLAS entry
  moved to Graduated; parent campaign entry synced; `goals/INDEX.md` regenerated
  (`goals doctor`: no new blocking findings).

## Review hardening

A three-lens adversarial Opus 5 panel (decision fidelity / repo grounding / packet
contract conformance) returned 17 verified findings, all folded in before publish:

- Restored dropped BRIEF semantics verbatim: application-identity prohibitions (never
  `OfficeAction` free-text, never a `PatentAsset` fixture key, never ST.13-only); both
  judgment slots (Rule 56 vs litigation-frame) representable from rung 1 with only the
  vocabulary deferred; the T2-F2 separate-fields clause.
- Gate-shape narrowing recorded in DECISIONS: the BRIEF's three lawful cross-slice
  shapes narrowed to the two `standards/ARCHITECTURE.md:632-636` sanctions (emitted
  events preferred; promoted `shared/use-cases` contract), final pick at goal P0 with
  an `architecture-guardian` check; app-level composition demoted to wiring-only.
- Added the missing `packages/shared/domain` EntityId registrations
  (`PatentCitationEventId`/`CandorDispositionId`, `PriorArtReferenceId` precedent) to
  target surfaces so GOAL.md's stop rule cannot forbid required work.
- Corrected the PR #557 refinement count (four, not five) in three surfaces;
  back-ported the #557 refinements into the exploration SOURCES ledger (EvidenceSpan
  superseded; ExecutionLedger attribution); fixed the one broken ATLAS relative link.

## Verification

- `bun run beep yeet verify` full local proof (this PR's publish path).
- `jq .` on both manifests; `goals doctor` green; `goals index --check` parity;
  GOAL.md launcher under the 4,000-char hard max; `git diff --check` clean.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

https://claude.ai/code/session_01KEpZ5d7GgoioRnD8kFJu7Z

---
<a href="https://app.blacksmith.sh/beep-effect/codesmith/beep-effect/pr/560"><picture><source media="(prefers-color-scheme: dark)" srcset="https://pr-comments-assets.blacksmith.sh/codesmith/view-with-codesmith-dark-v2.svg"><source media="(prefers-color-scheme: light)" srcset="https://pr-comments-assets.blacksmith.sh/codesmith/view-with-codesmith-light-v2.svg">View with [code]smith</picture></a> <a href="https://backend.blacksmith.sh/track/enable-autofix?expires=1788455665&installation_model_id=424656&pr_number=560&repository=beep-effect%2Fbeep-effect&return_to=https%3A%2F%2Fgithub.com%2Fbeep-effect%2Fbeep-effect%2Fpull%2F560&signature=9fb0cf267046e4a74a980a22480b56b62746732a20a34468fa82b74507efc0ba"><picture><source media="(prefers-color-scheme: dark)" srcset="https://pr-comments-assets.blacksmith.sh/codesmith/autofix-with-codesmith-dark.svg"><source media="(prefers-color-scheme: light)" srcset="https://pr-comments-assets.blacksmith.sh/codesmith/autofix-with-codesmith-light.svg">Autofix with [code]smith</picture></a>
<sup>Need help on this PR? Tag <code>@codesmith-bot</code> with what you need. Autofix is disabled.</sup>

### CODE REVIEW

File: goals/patent-citation-candor-gate/SPEC.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+
+Gated criteria (activate when the owning goal lands; never block this goal):
+
+- [ ] GATED on `uspto-prosecution-read`: bind the USPTO observation identity
+      (parser/vocabulary versions, checksums, freshness) and consume the live
+      quarantine producer.
+- [ ] GATED on `citation-extraction-engine`: the `CitationMention` handoff
+      into event discovery-provenance.
+- [ ] GATED on `agentic-professional-runtime`: compose a release-capable gate
+      decision vocabulary when one exists.
```
[the remark this answers is not in the document]
Comment (kriegcloud) [reply goals/patent-citation-candor-gate/SPEC.md]: PARTIAL. Verified: `RuntimeApprovalDecision` is `LiteralKit(["pending"])` (ProfessionalRuntime.values.ts:139) and `RuntimeCandidateLifecycle` is `["candidate"]` — the runtime cannot express advancement at all, so a blocking release-capable dependency would either block this goal on `agentic-professional-runtime` or widen the agents vocabulary; both are locked no-gos (SPEC.md:92-93, SPEC.md:198). Your second branch is correct, and the kernel is real: the rung-1 and rung-2 criteria are phrased entirely in blocking terms, so a green packet never proves release. Narrowing applied — the Objective, the rung-1 test criterion, and the rung-2 live-path criterion now state that release is proven at the `CandorPolicy` predicate boundary (law-practice owns blocked/released), with candidate advancement gated.

Folded in 19e6ecf9dc.

File: explorations/patent-citation-candor-gate/MAP.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+2. **Rung 2 — durability + live invocation** (`law-practice/domain` +
+   `tables` + `server`, plus db-admin). Durable storage for events,
+   dispositions, and the append-only IDS fact records on the `ExecutionLedger`
+   precedent end to end; the law-practice slice's **first** db-admin migration
+   with its PGlite migration test, both registered in `AcceptedProofManifest`;
+   and the filing-promotion path actually consulting `CandorPolicy` through
+   a doctrine-sanctioned cross-slice shape. Decompose narrows the BRIEF's
+   list to the two shapes `standards/ARCHITECTURE.md:632-636` sanctions —
+   emitted events (preferred) or a contract promoted into `shared/use-cases`
+   through the normal promotion gate; app-level entrypoint composition is
```
[the remark this answers is not in the document]
Comment (kriegcloud) [reply explorations/patent-citation-candor-gate/MAP.md:46]: PARTIAL. Verified: `packages/shared/use-cases` does not exist (ARCHITECTURE.md:352-356), SPEC.md:94 forbids new packages, and `RuntimeApprovalGate`/`RuntimeCandidateDraft` have no caller anywhere in `packages/` or `apps/`. The kernel is real and larger than stated: no product slice emits domain events either, so both sanctioned shapes are substrate-free today. Rejected the proposed fix — the binding shape pick is a locked P0 decision with an architecture-guardian check, so naming agents/apps surfaces now pre-empts it and grows scope past the two-week appetite. Folded instead: SPEC Target Surfaces names the consumer contingency and substrate state; Non-Goals carries a bounded Exception-Ledger carve-out for the promoted-contract branch; PLAN P0 must name the files its pick writes; MAP records the risk.

Folded in 19e6ecf9dc.

File: goals/patent-citation-candor-gate/SPEC.md [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+      union, `SourceTextIdentity` composition, receipt-based grounding, and
+      the two separately-triggered explicit states (staleness, quarantine)
+      that never rewrite evidence.
+- [ ] `CandorPolicy` `Context.Service` contract in `law-practice/use-cases`
+      owns the derived, fail-closed predicate with no stored closure state.
+- [ ] `CandorPolicy.test.ts` — written failing first, then green — proves:
+      promotion blocked until an attorney disposition covers the AI event's
+      exact observation version; a newer observation of the same source
+      re-blocks; the superseded event stops blocking only once the newer
+      event is dispositioned; quarantined and possible-duplicate fixtures
```
[the remark this answers is not in the document]
Comment (kriegcloud) [reply goals/patent-citation-candor-gate/SPEC.md]: PARTIAL. Verified: no attorney/practitioner role exists in the repo (`Membership.Role` is `owner`/`member` only), and `RuntimePrincipalId` is an unconstrained `S.NonEmptyString` (ProfessionalRuntime.contracts.ts:83). But a forged `RuntimePrincipalId` cannot release this gate — the disposition is a law-practice `BaseEntity`, which already carries `createdByPrincipal: Principal`, a union discriminating User/Agent/ServiceAccount/System (BaseEntity.ts:82, Principal.ts:244), and the agents-slice reviewer id is never the disposition author. The kernel stands: nothing constrained who may dispose. Bounded fix landed — SPEC now fails closed on principal kind (only `User` covers), adds the agent-authored negative test case, and states the trust boundary; MAP gains a capability row plus a practitioner-authority follow-on. A new authorization service is scope growth against locked decision 3.

Folded in 19e6ecf9dc.

File: goals/patent-citation-candor-gate/SPEC.md [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+  `agents/use-cases` import is forbidden.
+- Quarantine has no live producer at rung 1: prove the predicate's quarantine
+  branch against hand-constructed fixtures; the live producer is gated on
+  `uspto-prosecution-read` landing raw-preserving unknown-code failures.
+- Possible duplicates are treated as undisposed, never resolved — the
+  reconciliation inbox is a named follow-on, not this goal.
+- A persisted `TextAnchorVerificationReceipt` is not live proof;
+  re-verification via `verifyTextAnchor` is required before any "current"
+  claim, and "current" is defined against source-version identity, never
+  receipt existence.
```
[the remark this answers is not in the document]
Comment (kriegcloud) [reply goals/patent-citation-candor-gate/SPEC.md:172]: PARTIAL. Real kernel: SPEC:140-143 mandates re-verification via `verifyTextAnchor` (VerifiedTextAnchor.ts:120-131, plus `Crypto.Crypto` at :351), yet no packet doc named where canonical source text comes from. The proposed fix over-reaches. `SourceTextResolver` already exists (`file-processing/src/SourceText/index.ts:357,374`), and `@beep/law-practice-use-cases` already depends on `@beep/file-processing` (package.json:52), which ARCHITECTURE:622-623 permits — a new service would duplicate a live brick. The dichotomy is also wrong: unresolvable sources are fail-closed undisposed events (decision 3), and rung 1 is in-memory only. Edits name the existing port in Target Surfaces, Constraints, Non-Goals, and MAP, add a fail-closed resolution test case, and bind the live layer at app wiring, gated on `uspto-prosecution-read`.

Folded in 19e6ecf9dc.

File: goals/patent-citation-candor-gate/SPEC.md [hunk 5]
```diff
@@ -0,0 +1,241 @@
...
+      stay uncovered; examiner-observed events record without gating.
+- [ ] The test runs slice-isolated: in-memory/test-only layers, no other
+      slice booted, no app runtime Layer.
+
+Rung 2 — durability + live gate:
+
+- [ ] Durable storage for events, dispositions, and append-only IDS fact
+      records via ports → repo/layer on the `ExecutionLedger` precedent.
+- [ ] The law-practice slice's first db-admin migration with its PGlite
+      migration test, both registered in `AcceptedProofManifest`.
```
[the remark this answers is not in the document]
Comment (kriegcloud) [reply goals/patent-citation-candor-gate/SPEC.md]: FOLD. Verified: the rung-2 criterion scopes "append-only" to IDS fact records only (SPEC.md:184-185), law-practice entities inherit BaseEntity's `rowVersion`/`updatedAt`/`updatedByPrincipal` (PriorArtReference.model.ts:50, BaseEntity.ts:84-88), and no criterion or constraint stops a disposition being overwritten — while SPEC.md:11-13 makes that row the sole record of judgment. BRIEF/DECISIONS are silent on revision semantics; every "supersede" there concerns observation versions, not dispositions. The fix strengthens the locked fail-closed and fact/judgment decisions and costs almost nothing: ClaimDisposition.ports.ts:131-135 already sets the append-and-read-only port shape, and the evidence-verification migration (:36-46) shows the trigger on a BaseEntity table. Four bounded SPEC edits added: lifecycle slot at rung 1, a withdrawn-disposition test case, append-only ports, and trigger plus PGlite assertion in the already-required migration.

Folded in 19e6ecf9dc.

File: explorations/patent-citation-candor-gate/MAP.md [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+activate whenever their owning goals land — they gate acceptance criteria,
+never the rungs' start.
+
+## First Vertical Slice
+
+The failing `CandorPolicy.test.ts` in `packages/law-practice/use-cases`: one
+`PatentReference` with an examiner-observed event and an AI-discovered event;
+assert filing promotion stays blocked until an attorney disposition covers
+the AI event's exact observation version, then flips blocked again when a
+newer observation version of the same source arrives (staleness re-blocks:
```
[the remark this answers is not in the document]
Comment (kriegcloud) [reply explorations/patent-citation-candor-gate/MAP.md:63]: FOLD. Verified: MAP.md:59-64 and SPEC.md:173-178 hold only one *current* AI-discovered source at a time — the newer observation supersedes the older — so the universal quantifier in SPEC.md:15 and BRIEF.md:107 is never exercised above cardinality one. A version-aware existential predicate (release iff some current event has an exact-version disposition) satisfies every assertion in that sequence. The cruder "any disposition releases" reading is already caught by the re-block step, and "possible-duplicate fixtures stay uncovered" proves a per-event fail-closed state, not universality. Fix: a second `PatentReference` with its own current AI event in the rung-1 fixture — dispose one, still blocked; dispose both, released. Added to MAP.md First Vertical Slice and the SPEC rung-1 criterion. No locked decision touched.

Folded in 19e6ecf9dc.

File: goals/patent-citation-candor-gate/SPEC.md [hunk 7]
```diff
@@ -0,0 +1,241 @@
...
+   representable from rung 1 without deriving either.
+3. **Derived gate, no stored closure** — no "duty satisfied" state exists
+   anywhere; the predicate is recomputed from events + dispositions; stale,
+   quarantined, and possible-duplicate events count as undisposed (fail
+   closed).
+4. **Predicate semantics** — coverage requires a disposition bound to the
+   exact observation version AND that version being current for its source; a
+   superseded event stops blocking only once the newer event is itself
+   dispositioned; examiner events record without gating (widening the
+   quantified set is a later align question).
```
[the remark this answers is not in the document]
Comment (kriegcloud) [reply goals/patent-citation-candor-gate/SPEC.md:47]: PARTIAL. Verified: `SourceTextIdentity` (SourceTextIdentity.ts:119-145) carries digests and pinned extractor/normalization versions — no revision order, parent, or head marker — and nothing in SPEC/MAP/BRIEF/DECISIONS says how "current" is established. Real gap.

Not folded: an authoritative lineage/head mechanism. That binding is already the gated criterion on `uspto-prosecution-read` (SPEC.md:200-202), whose objective owns retrieval time, freshness, and cursor/upstream identity; building a comparator here forks that SPEC.

Bounded fix: a SPEC constraint making currency declared, never inferred — explicit supersession naming the exact prior version, lineage written once, head derived (`EdgeVersion.supersedesId` posture, pattern only), absent/ambiguous/forked lineage failing closed. Plus out-of-order and replay fixtures on the rung-1 test criterion, and a corrected MAP capability row.

Folded in 19e6ecf9dc.

**Comment by kriegcloud** [comment]
**Re: "Rung-two package contradiction" (Confidence Score 4/5 blocker)** — the finding rests on a stale knowledge-base snapshot. Both packages exist on disk at the reviewed commit:

```
$ ls packages/law-practice/
domain  server  tables  use-cases
```

`packages/law-practice/tables` and `packages/law-practice/server` are live workspace packages, so rung 2's target surfaces name existing packages and the "No new packages" Non-Goal is satisfiable as written — no contradiction.

The genuinely absent package is `packages/shared/use-cases` (the promoted-contract gate shape's home, which `standards/ARCHITECTURE.md:352-356` itself calls "future"). Commit 19e6ecf9dc2a89a0fd790d4503f9450cb7911281 makes that contingency explicit: the Non-Goal now carries a bounded Exception-Ledger carve-out for the promoted-contract branch, and the P0 exit criteria require the chosen shape to name the exact consumer/wiring files it writes before P2 starts. The same commit folds the adjudicated refinements from the seven Codex P1 review threads (each thread carries its own verdict reply).
